packages/espresso/android/src/main/java/androidx/test/espresso/flutter/action/FlutterViewAction.java

```diff
@@ -47,7 +47,7 @@
  * <p>This class acts as a bridge to perform {@code WidgetAction} on a Flutter widget on the given
  * {@code FlutterView}.
  */
-@Beta
+@ExperimentalTestApi
```

Comment:
Is it possible to move this annotation to methods instead of the whole class?

Reply:
Done